impls/codec/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015-2017 Parity Technologies
```

Comment:
All the copyright years should be `2015-2018`.

Reply:
I changed all touched files in this PR to be `2015-2018`, but left everything else unchanged, to avoid making this PR too big, and also remembered this was the guideline for changing copyright years.